openwisp_controller/connection/tests/test_api.py

```diff
@@ -387,6 +442,61 @@ def test_create_command_without_connection(self):
         )
 
 
+# The same tests, but with a normal user
+class TestCommandsApiNonAdmin(TestCommandsAPI):
```

Comment:
@pandafy aren't the tests in this class now redundant with `test_endpoints_for_org_operators_own_org` above?